src/state-summary/state-card-input_text.html

```diff
@@ -0,0 +1,92 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+
+<link rel="import" href="../../bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+<link rel="import" href="../../bower_components/paper-input/paper-input.html">
+
+<link rel="import" href="../components/entity/state-info.html">
+
+<dom-module id="state-card-input_text">
+  <template>
+    <style is="custom-style" include="iron-flex iron-flex-alignment"></style>
+    <style>
+      paper-input {
+        margin-left: 16px;
+      }
+    </style>
+
+    <div class='horizontal justified layout'>
+      <state-info state-obj="[[stateObj]]" in-dialog='[[inDialog]]'></state-info>
+      <paper-input
+        minlength='[[min]]' maxlength='[[max]]' value='{{value}}' pin
+        disabled='[[disabled]]' auto-validate pattern='[[pattern]]'
+        on-change='selectedValueChanged' on-tap='stopPropagation'>
+      </paper-input>
+    </div>
+  </template>
+</dom-module>
+
+<script>
+Polymer({
```

Comment:
I know that you probably copy pasted this from other components, but we're in the process of migrating to Polymer 2 classes for components. We should no longer add Polymer 1 components.

See [this component](https://github.com/home-assistant/home-assistant-polymer/blob/206f624fb6008f19ae1c514a16ce77461a9cfbcd/src/state-summary/state-card-content.html) for an example of a Polymer 2 class. The conversion is very straight forward.

Reply:
Yes, it started as a copy of input_slider. I'll figure out the changes and I guess do the same to input_slider later, since it should be nearly identical changes to make, when I do the rename to input_number.